modules/azure/src/main/java/org/testcontainers/containers/AzuriteContainer.java

```diff
@@ -0,0 +1,529 @@
+        this.version = true;
+        return this;
+    }
+
+    /**
+     * Enables OAuth authentication. Requires SSL as well.
+     * {@link #withSsl(File, String)}
+     * {@link #withSsl(File, File)}
+     *
+     * @param oauth The OAuth parameter
+     * @return this
+     */
+    public AzuriteContainer withOauth(final String oauth) {
+        this.oauth = oauth;
+        return this;
+    }
+
+    /**
+     * Tells Azurite to skip checking the API version.
+     *
+     * @return this
+     */
+    public AzuriteContainer withSkipApiVersionCheck() {
+        this.skipApiVersionCheck = true;
+        return this;
+    }
+
+    /**
+     * Tells Azurite to disable product style URLs.
+     *
+     * @return this
+     */
+    public AzuriteContainer withDisableProductStyleUrl() {
+        this.disableProductStyleUrl = true;
+        return this;
+    }
+
+    /**
+     * Tells Azurite to store content in memory using the default limits.
+     *
+     * @return this
+     */
+    public AzuriteContainer withInMemoryPersistence() {
+        this.inMemoryPersistence = true;
+        return this;
+    }
+
+    /**
+     * Tells Azurite to store content in memory using a custom limit.
+     *
+     * @param limit The memory limit in MBs
+     * @return this
+     */
+    public AzuriteContainer withInMemoryPersistence(final long limit) {
+        this.inMemoryPersistence = true;
+        this.extentMemoryLimit = limit;
+        return this;
+    }
+
+    /**
+     * Adds an account name and account key to be able to use them for authentication.
+     * (default name: {@link #WELL_KNOWN_ACCOUNT_NAME})
+     * (default key: {@link #WELL_KNOWN_ACCOUNT_KEY})
+     *
+     * @param accountName The name of the account
+     * @param primaryKey  The account key
+     * @return this
+     */
+    public AzuriteContainer addAccountCredentials(final String accountName, final String primaryKey) {
+        return addAccountCredentials(accountName, primaryKey, null);
+    }
+
+    /**
+     * Adds an account name and account keys to be able to use them for authentication.
+     * (default name: {@link #WELL_KNOWN_ACCOUNT_NAME})
+     * (default key: {@link #WELL_KNOWN_ACCOUNT_KEY})
+     *
+     * @param accountName  The name of the account
+     * @param primaryKey   The primary account key
+     * @param secondaryKey The secondary account key (optional)
+     * @return this
+     */
+    public AzuriteContainer addAccountCredentials(
+        final String accountName,
+        final String primaryKey,
+        final String secondaryKey
+    ) {
+        final StringBuilder credentialBuilder = new StringBuilder().append(accountName).append(":").append(primaryKey);
+        Optional.ofNullable(secondaryKey).ifPresent(s -> credentialBuilder.append(":").append(s));
+        this.azuriteAccounts.add(credentialBuilder.toString());
+        return this;
+    }
+
+    @Override
+    protected void configure() {
+        super.configure();
```

Comment:
```suggestion

```

Reply:
Thank you! Just out of curiosity, what is the value of not calling super? I have found a couple of examples in the repo where super is called and it made sense thinking that one day the super implementation might add some common code and every module calling the super will benefit from it automatically.

https://github.com/search?q=repo%3Atestcontainers%2Ftestcontainers-java%20super.configure()%3B&type=code

Reply:
Okay, I get it, thank you! I was just worried because we are breaking the Liskov substitution principle by doing so...